ui/components/ui/update-nickname-popover/update-nickname-popover.stories.js

```diff
@@ -1,45 +1,61 @@
 import React, { useState } from 'react';
-import { action } from '@storybook/addon-actions';
-import { text } from '@storybook/addon-knobs';
 import Button from '../button';
+import README from './README.mdx';
 import UpdateNicknamePopover from '.';
 
 export default {
   title: 'Components/UI/UpdateNickname',
   id: __filename,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    address: { control: 'text' },
+    onClose: { action: 'Close Update Nickname Popover' },
+    onAdd: { action: 'Submit Nickname' },
```

Comment:
I think we could adopt a convention where we keep the action name the same as the prop so the developer knows exactly what prop is being passed to

```suggestion
    onClose: { action: 'onClose' },
    onAdd: { action: 'onAdd' },
```